tests/Integration/Framework/Test_ProxyDB.py

```diff
@@ -0,0 +1,280 @@
+""" This is a test of the ProxyDB
+
+    It supposes that the DB is present and installed in DIRAC
+"""
+
+# pylint: disable=invalid-name,wrong-import-position,protected-access
+import os
+import sys
+from sqlalchemy import create_engine
+import unittest
+
+from DIRAC.Core.Base.Script import parseCommandLine
+parseCommandLine()
+
+from DIRAC import gLogger, gConfig
+from DIRAC.Core.Utilities.CFG import CFG
+from DIRAC.FrameworkSystem.DB.ProxyDB import ProxyDB
+from DIRAC.ConfigurationSystem.Client.Utilities import getDBParameters
+
+# Create needed for tests rows
+result = getDBParameters('Framework/ProxyDB')
+if not result['OK']:
+  raise RuntimeError('Cannot get database parameters: %s' % result['Message'])
+dbParam = result['Value']
+engine = create_engine('mysql://%s:%s@%s:%s/%s' % (dbParam['User'],
+                                                   dbParam['Password'],
+                                                   dbParam['Host'],
+                                                   dbParam['Port'],
+                                                   dbParam['DBName']))
+connection = engine.connect()
```

Comment:
I don't get why you are using sqlalchemy here.

Reply:
not use now